File: encapsulation-ruby/test/level_0/no_encapsulation_test.rb
require "test/unit"
require "level_0/no_encapsulation"

class NoEncapsulationTest < Test::Unit::TestCase

  # How would we deal with more than one galleon?
  def test_can_determine_fighting_strength    
    fighting_strength = 2;
    assert(fighting_strength, 2)
  end
end

File: encapsulation-csharp/src/test/csharp/encapsulation/level_0/noEncapsulationTest.cs
namespace encapsulation.level_0 {
    using NUnit.Framework;

    [TestFixture]
    public class NoEncapsulationTest
    {

      [Test]
        public void CanCompareForEquality()
        {
            Assert.AreEqual(4, 4);
        }
    }
}